languages/fr/assess-api.json

```diff
@@ -26,37 +26,37 @@
     "retry": "Réessayer",
     "save": "Enregistrer",
     "saveButtonLabel": "Quitter", 
-    "saveError": "Erreur lors de l/'enregistrement.",
+    "saveError": "Erreur lors de l'enregistrement.",
     "saveInProgress": "Enregistrement...",
     "savingInProgress": "Enregistrement en cours.",
-    "saving": "l/'enregistrement",
-    "saved": "enregsitrée",
-    "saveTestSuccessfully": "Votre activité a bien été enregistrée",
-    "saveTestSuccessfullyPrompt": "Souhaitez-vous continuer l/'{{test}}, ou quitter ?",
-    "resume": "reprendre",
+    "saving": "l'enregistrement",
+    "saved": "enregistrée",
```

Comment:
This one, not knowing the context, could be both `enregistré` or `enregistrée`

Reply:
This is an annoying one... I don't know when this label is used, and looking at the rest of the file, I'm not sure it's used. Nontheless, Angélique had it like that, I was only fixing a typo, and it's likely that the `{{test==activité}} a été {{saved==enregistrée}}`. So I agree it's potentially incorrect, but I think it's safe at the moment.